docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,122 @@
+
+2. Enter your **Test Name**. Project tests are saved to a tests SQL file by default.
+
+3. Click **OK** to create your new test.
+
+4. Create your test Model by dragging and dropping Gems to your Visual canvas. Connect them to your Data Test. You can also write your test on the Code view. You can use as many Models and other database objects as you’d like in your test.
+
+### Run a test
+
+After you’ve developed your project test and executed all of your Models, you can run the test.
+
+1. Click the **Play** button on either the canvas or an individual Gem to execute your test. The outputted table from the Gem is what’s tested.
+   1. Run a whole test. Clicking the canvas **Play** button executes the complete test and shows the interim data after the Data Test Gem. Additionally, it displays the final test status, succeeded, warning, or failed, in a summary.
+   2. Run a partial test. Clicking the Gem **Play** button executes the test SQL only for a particular Gem, and displays that Gem’s data on the output. This option doesn’t execute the data test.
+2. Click **See Run Details** to view the test Summary. Depending on the outcome of the test, the icon displays a different color.
+
+   ![See Run Details](img/project-test-run-details.png)
+
+   You can click to expand the test logs in the Summary to view the dbt logs.
+
+   ![View test summary](img/project-test-summary.png)
+
+In addition to the previous relational integrity test, you can create tests to check that the total payment amount from customers is positive or simply check that all of your tables are still accessible.
+
+## Fix a failed test
+
+If your project test fails, check your test Model for any changes.
+
+![Failed test](img/project-test-fail.png)
+
+Some examples of test failure causes include:
+
+- Happy path: There is now a row in the table → you have detected the thing you were trying to detect!
+- The table no longer exists since one or more input data sources were deleted or inaccessible
+- Dataset schema changes, such as in the column or datatype
+- The seed in the test is different from what’s in your warehouse
+- Code releases that caused a breaking change
+
+Note: Make sure you have write permission to create a new table in your data warehouse, otherwise you may run into errors while trying to run your own tests.
+
+By default, project tests are configured to fail if the table has one or more rows. You can also modify the passing condition of your test through advanced options.
+
+## Configure a test
+
+You can configure your own passing condition of your project test.
+
+You can use the configurations to help decide which tables to focus on if you have too many with failed rows. Or conversely, you might care less about cases with multiple failed rows. For example, you might have a customer churn use case where you have 50 complaints from one customer. Configuring your test can help you identify and take action, such as assigning more resources to assist that customer.
```

Comment:
"which tables to focus on"
This sounds incorrect. The project test is ONLY testing a single table as far as I understand.

Reply:
Right, I updated the text to "cases".

Reply:
Aha, ok "cases" works! 

But the second suggestion is still relevant. In particular, you care MORE rather than LESS.